ubccr/coldfront
# Fixes for "Update Allocation Form Does Not Update" Bug

Testing instructions:

- Deploy locally or to QA.
- Create allocation(s) as needed with data that will be updated
- Navigate to "View All Allocations," then to a previously-created allocation
- Modify one of its attributes (e.g. ITSD Ticket)
- Submit updates and observe the re-loading of the form.

When the update has been submitted, an Allocation Change Request should have been created.  It can be accessed by navigating to Admin -> Allocation Change Requests.  The change request can be optionally approved to complete the update to the data, but the existence of the change request reflects a successful test of this PR.
